tests/test_lists2safebrowsing.py

```diff
@@ -188,37 +194,39 @@
 )
 
 PROCESS_PLUGIN_BLOCKLIST_EXPECTED_OUTPUT_WRITES = (
-    b"a:%d:32:64\n",
+    b"a:%d:32:96\n",
     (
         DUMMYTRACKER_DOMAIN_HASH,
-        (b"\xbc\x9a\x8f+o\xff\xd5\x85q\xe1\x88\xbb\x11\x05E\xf8\xfb:"
-         "\xf5\x1c\xdf\x1acimPZ\x98p\xa8[\xe5"),
+        EXAMPLE_DOMAIN_HASH,
+        GOOGLE_DOMAIN_HASH,
     ),
 )
 
 PROCESS_PLUGIN_BLOCKLIST_EXPECTED_LOG_WRITE_INFO = (
     (CANONICALIZE_TESTCASES[0][1], CANONICALIZE_TESTCASES[0][2],
         "e5a907c8ff3672a9cbc8f1d3a2110c5cbe7fdb31bb5edf44bc58a8f1553b23e2"),
-    (CANONICALIZE_TESTCASES[1][1], CANONICALIZE_TESTCASES[1][2],
+    ("example.com/", "example.com/",
+        "73d986e009065f182c10bcb6a45db3d6eda9498f8930654af2653f8a938cd801"),
+    ("https://www.google.com", "www.google.com/",
         "bc9a8f2b6fffd58571e188bb110545f8fb3af51cdf1a63696d505a9870a85be5"),
 )
 
 GET_TRACKER_LISTS_TESTCASES = (
     (
         "default", "tracking-protection",
-        {"adnetwork.net", "appcast.io", "clickguard.com",
-         "google-analytics.com", "postrank.com", "twimg.com",
-         "twitter.com", "twitter.jp"}
+        {"adnetwork.net/", "appcast.io/", "clickguard.com/",
```

Comment:
was there something that broke when the trailing `/` was removed in this PR? Or just following the form the domains on shavar-prod-lists have it as?